yew-macro/tests/derive_variants_test.rs

```diff
@@ -0,0 +1,7 @@
+#[allow(dead_code)]
+#[rustversion::attr(stable(1.45), test)]
+fn tests() {
+    let t = trybuild::TestCases::new();
+    t.pass("tests/derive_variants/pass.rs");
+    t.compile_fail("tests/derive_variants/fail.rs");
```

Comment:
Looks like there's no `stderr` file yet... Please run the tests locally to generate it